fv/run-batches

```diff
@@ -16,7 +16,7 @@
 
 : ${FV_NUM_BATCHES:=4}
 : ${FV_BATCHES_TO_RUN:=1 2 3 4}
-: ${FV_SLOW_SPEC_THRESH:=90}
+: ${FV_SLOW_SPEC_THRESH:=300}
```

Comment:
I feel a little worried this will encourage people to write tests with massive sleeps in.  What's the harming in keeping the "SLOW TEST" output as we currently have it?

Reply:
I was just making the output a bit messy.  When the test monitor thread is grepping for the start of the test with `-C 10` it ends up printing half a SLOW test warning half of the time.  Will revert.